com.ibm.streamsx.topology/opt/python/packages/streamsx/topology/context.py

```diff
@@ -450,3 +453,4 @@ class ConfigParams:
     """
     VCAP_INFO = 'topology.service.vcap'
     VCAP_NAME = 'topology.service.name'
+    REMOTE_SUBMIT = 'topology.remote_submit'
```

Comment:
Please be consistent with the existing properties which use camel case.

Should we change this to `topology.forceRemote` so that it can be used with other contexts?